tools/scoop-kopia.json.template

```diff
@@ -1,15 +1,15 @@
 {
     "version": "VERSION",
+    "description": "Fast and secure open source backup.",
+    "homepage": "https://kopia.io/",
+    "license": "Apache-2.0",
     "architecture": {
         "64bit": {
             "url": "https://github.com/SOURCE_REPO/releases/download/vVERSION/kopia-VERSION-windows-x64.zip",
             "bin": [
-                "kopia-VERSION-windows-x64/kopia.exe"
+                "kopia.exe"
             ],
             "hash": "HASH_WINDOWS_AMD64"
         }
-    },
-    "homepage": "https://kopia.io/",
-    "license": "Apache-2.0",
-    "description": "Fast and secure open source backup."
```

Comment:
I took a look at scoop tutorial - could we add `"extract_dir": "kopia-VERSION-windows-x64"` ? I really would like to avoid changing the zip file structure as it has benefits.

Reply:
You are right. Let's do that.